.gitignore

```diff
@@ -1,5 +1,6 @@
 /target
 /.envrc
+/.direnv
```

Comment:
What's this? [direnv](https://direnv.net/) expects `.envrc` or `.env`.

Reply:
https://github.com/nix-community/nix-direnv produces derivations for the shell.